remove the use of depreceated pacakge ioutil and comply with gopls linter

bokunodev opened the pull request:
Changes to be committed:
	modified:   golden_test.go - remove ioutil pacakge
	modified:   init_test.go - remove ioutil pacakge
	modified:   licenses.go - formating and change fmt.Errorf to errors.New to comply with gopls linter

bokunodev committed:
remove the use of depreceated pacakge ioutil

Changes to be committed:
	modified:   golden_test.go - remove ioutil pacakge
	modified:   init_test.go - remove ioutil pacakge
	modified:   licenses.go - formating and change fmt.Errorf to errors.New to comply with gopls linter

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/spf13/cobra-cli?pullRequest=113) <br/>All committers have signed the CLA.

bokunodev committed:
fix compareFiles, fix test

Changes to be committed:
	modified:   golden_test.go - fix compareFiles function, replace year in golden test file to current year
